### ISSUE TITLE
Kompile does not accept --transition, --superheat, or --supercool

### ISSUE THREAD

**Issue by andreistefanescu** [kframework/k-legacy#443]
Try in the unified builtins branch
kompile imp.k --transition lookup increment assignment read print
kompile imp.k --superheat division --supercool lookup increment

**Comment by dwightguth** [comment]
This is because only the first is interpreted as a tag and the rest become
files to kompile, of which you are not allowed to have more than one.
Currently, the correct syntax for what you want to do uses quotation marks.
Argv should be "imp.k", "--transition", " lookup increment assignment read
print"; not "imp.k", "--transition", " lookup", "increment", "assignment",
"read", "print".

I think I know a way to make it act like you describe though, which would
be a cleaner implementation, so I will get to this tomorrow.

**Comment by andreistefanescu** [comment]
The problem is that this is what ktest generates. I'm ok with using "", as long as ktest is fixed.

**Comment by dwightguth** [comment]
Oh. No, ktest generates it correctly, it just doesn't print the separation
on stdout. That's a separate issue.

**Comment by dwightguth** [comment]
This is three separate issues. The first is addressed by #445, the second by #446 , and the third (to make --transition be variable arity instead of using a converter) can't be done easily right now without a larger change to ktest, because it has the property that a single option can span across multiple words. So I think the thing to do is to leave this open and make it depend on #392.

**Comment by dwightguth** [comment]
I decided against using variable arity because it has issues encoding certain strings. Resolving this.
